src/uwtools/tests/utils/test_api.py

```diff
@@ -118,20 +118,19 @@ def test_str2path_convert():
     assert result == Path(val)
 
 
-def test__execute(execute_kwargs, tmp_path):
-    config = tmp_path / "config.yaml"
-    with open(config, "w", encoding="utf-8") as f:
-        yaml.dump({"some": "config"}, f)
```

Comment:
Another case (see some previous PR...) where we don't actually need the config file on disk; we can provide it as an in-memory disk and do a little less filesystem IO.